wasm-node/javascript/test/misc.mjs

```diff
@@ -40,14 +40,33 @@ test('malformed JSON-RPC request generates an error', async t => {
     .then(() => client.terminate());
 });
 
+test('invalid JSON-RPC method generates an error', async t => {
+  const client = start({ logCallback: () => { } });
+  await client
+    .addChain({ chainSpec: westendSpec })
+    .then((chain) => {
+      chain.sendJsonRpc('{"jsonrpc":"2.0","id":1,"method":"this method doesnt exist","params":[]}');
+      return chain;
+    })
+    .then(async (chain) => {
+      const response = await chain.nextJsonRpcResponse();
+      const parsed = JSON.parse(response);
+      if (parsed.id === null && parsed.error)
```

Comment:
Actually, this shouldn't be `null`, right? It should be the `id` sent.

Reply:
Indeed. The tests take a very long time to run for me locally (several minutes), and I ran out of patience before it hit the new test.

Reply:
I understand the feeling, our zombienet tests have the same problem :)